new_dawn_server/urls.py

```diff
@@ -43,4 +49,4 @@
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^api/', include(v1_api.urls)),
-]
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
@duckmoll Notice that I just removed STATIC_ROOT from here. That dir contains css and js files (for admin tool) which we might not need user to be accessable.

Reply:
kk